parsuricata/_parser.py

```diff
@@ -91,7 +89,7 @@
     option: KEYWORD ";"
           | KEYWORD ":" settings ";"
 
-    KEYWORD: /[a-z_]+/i
+    KEYWORD: /[a-z_\.]+/i
```

Comment:
When inside a character group, the only thing that ever needs to be escaped is the closing bracket. That backslash can get goooone
```suggestion
    KEYWORD: /[a-z_.]+/i
```